script/functions.sh

```diff
@@ -82,7 +84,32 @@ function run_specs_and_record_done {
     rspec_bin=script/rspec_with_simplecov
   fi;
 
-  $rspec_bin spec --backtrace --format progress --profile --format progress --out $SPECS_HAVE_RUN_FILE
+  $rspec_bin spec --backtrace --profile --format progress --out $SPECS_HAVE_RUN_FILE
+}
+
+function run_sample_specs {
+  if [ ! -f ./smoke_specs/$SPECS_HAVE_RUN_FILE ]; then # don't rerun specs that have already run
+    pushd ./smoke_specs
+    echo
+    echo "Running smoke specs for $RSPEC_VERSION"
+    echo
+    unset BUNDLE_GEMFILE
+    bundle_install_flags=`cat ../.travis.yml | grep bundler_args | tr -d '"' | grep -o " .*"`
+    travis_retry bundle install $bundle_install_flags
+    cp ../.rspec .rspec
+    set +e
+    bin/rspec spec ../spec --format NyanCatFormatter --out $SPECS_HAVE_RUN_FILE
+    local status=$?
+    set -e
+    popd
+    if test $status = 42; then
```

Comment:
Nevermind, I see where you configure it below now.  I think a comment here explaining why 42 would help.